### DISCUSSION TITLE
our name

### DISCUSSION THREAD

**Discussion by pdurbin** [bostonopen/bostonopen.github.io#1] [General]
We've had [two](http://blog.greptilian.com/2022/12/10/open-source-meetup-in-brookline/) [meetings](http://blog.greptilian.com/2023/01/03/open-source-meetup-in-brookline-2/) so far. What do we call ourselves? 😄 🤔 

I went ahead and set up this @bostonopen org on GitHub as a first pass at having a name.

I jotted down almost 50 names before I settled on bostonopen but let's talk about it. Please add names to [my doc](https://docs.google.com/document/d/1QuMoIWqkZmNNdUCBHM6b10C5KKgEeslyVZXHxiY0WZA/edit?usp=sharing) or your own doc or wherever you like, such as in a comment below.

I'll say a bit more on my thought process for bostonopen.

First, Boston obviously has broader appeal than the suburb of Boston we've been meeting in. It's the capital of New England.

As for the word "open", my original idea was bostonopen.dev (which I actually did register today). bostonopen.dev conveys Boston, of course, but also "open" and something with development. For me "open source" is the obvious conclusion of what this domain might be about. That said, bostonopen.github.io conveys more or less the same thing. Boston, open, GitHub... this is probably an open source group in Boston. So either domain speaks to the target audience, in my mind.

The other nice thing about "open" is that the open source movement has spawned a number of related movements (or at least terms) such as open access, open science, open culture, open data, open government, open hardware, open research, and open education. I like all of these things too! So I like that bostonopen leaves things somewhat ambiguous. You want to talk about open culture? Sure, come on in! Open science? Yes! We love science!

I'll close with a couple possible logos.

Here's my original sketch:

![bostonopensketch](https://user-images.githubusercontent.com/21006/215298201-6819af43-2e3f-43ce-93cb-7dc3865938d8.jpg)

I went ahead and put "open source" in this one and set it as the org logo on GitHub for now: https://github.com/bostonopen/bostonopen.github.io/commit/39eebab833fc914598db1d12f05b3c47d3c2fb26

![logo](https://user-images.githubusercontent.com/21006/215298060-29bbcfe0-ffa0-4697-abf9-34118db9df8b.png)

My daughter created this one, based on the sketch above: https://github.com/bostonopen/bostonopen.github.io/commit/d4bdface6ed30b8c8bd29e9d4352779c30286dc0

![logo-box](https://user-images.githubusercontent.com/21006/215298114-35994e89-88e2-4939-9bfa-37de91b72047.png)

Here's my list of potential names, a snapshot of the doc above: [names.txt](https://github.com/bostonopen/bostonopen.github.io/files/10528737/names.txt)

**Comment by pdurbin** [comment]
> my original idea was [bostonopen.dev](https://bostonopen.dev/) (which I actually did register today)

I went ahead and set this up. Below is how it looks on mobile. What do you think?

![Screenshot_20230201-071459 copy](https://user-images.githubusercontent.com/21006/216045525-cc82b3f9-3a30-4b47-a531-8f644f681744.png)

**Comment by pdurbin** [comment]
I still don't know exactly what we're called.

At a [code and coffee](https://www.bostoncodeandcoffee.com) meetup on Saturday, I just said we're an open source meetup and our website is bostonopen.dev

So in practice our website is our name: Boston Open Dev or Boston Open for short, I suppose. @bostonopen on GitHub.

I dunno, what do other people think?

**Comment by pdurbin** [comment]
On Mastodon, thanks to @orblivion (see https://github.com/orgs/bostonopen/discussions/6 ), we are Boston Open Dev or `@bostonopen` on floss.social: https://floss.social/@bostonopen

![267806906-83be29d4-405a-401f-a56f-41dc329f3d95](https://github.com/bostonopen/bostonopen.github.io/assets/21006/125e6729-a7bc-435e-bd25-9da059ae5e45)

**Comment by orblivion** [comment]
I just copied the website :shrug:Message ID: <bostonopen/bostonopen.github.
***@***.***>

>
